tools/setup

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh
+
+ROOT=$(dirname $0)/..
+
+# 1. Install nvm
```

Comment:
Consider converting comments to status messages to let the person executing the script gauge progress.

Reply:
Good idea.